packages/sdk-node/src/Clerk.ts

```diff
@@ -50,21 +50,38 @@ export type RequireAuthProp<T> = T & {
   };
 };
 
-export type Middleware = (
+type MiddlewareWithAuthProp = (
   req: WithAuthProp<Request>,
   res: Response,
   next: NextFunction
 ) => Promise<void>;
 
-export type MiddlewareOptions = {
+type MiddlewareRequireAuthProp = (
+  req: RequireAuthProp<Request>,
+  res: Response,
+  next: NextFunction
+) => Promise<void>;
+
+export type Middleware = MiddlewareWithAuthProp | MiddlewareRequireAuthProp;
+
+export type MiddlewareOptionsInLooseMode = {
```

Comment:
❓ Same question as below, is this a public or private type?

Reply:
It's private. This is now fixed.